src/python/setup.py.in

```diff
@@ -13,7 +13,7 @@ class BinaryDistribution(Distribution):
 setup(
     name='@TARGET_NAME@',
     version='@VERSION_INFO@',
-    packages=find_packages(),
+    packages=['onnxruntime_genai', 'onnxruntime_genai.models'],
```

Comment:
Should this just be onnxruntime_genai?

Reply:
If `onnxruntime_genai.models` is not listed, then the `models` folder does not appear in the wheel.